.github/workflows/daily.yml

```diff
@@ -7,15 +7,15 @@ on:
 
 env:
   NODE_VERSION: lts/*
+  CC: clang
+  CXX: clang++
 
 permissions:
   contents: read
 
 jobs:
   build-lto:
-    runs-on: ubuntu-latest
-    # not working on gcc-8 and gcc-9 see https://github.com/nodejs/node/issues/38570
-    container: gcc:11
```

Comment:
this is for test lto on gcc. keep `container: gcc:11`

Reply:
> The long-term goal is to remove support for GCC, so keeping GCC is in conflict with it

gcc is very popular, I don't think remove gcc is a good idea. Even v8 accept patches make build works on gcc.
Some cpp feature also only not exists on clang.

The root requirements actually comes from community (build gcc with LTO).

If tsc decide remove gcc, I am okay with this. Other than I intend to keep this.

Reply:
I removed clang from this build.

Reply:
> I'm ok to move gcc-with-lto testing to Jenkins CI

This works in 3 years without any regression. So I guess it's okay for it in GA. I also lock the gcc major version.
If you intend to move it to Jenkins CI, it should also work.